iOS-Email-Client/Views/EmailTableViewCell.swift

```diff
@@ -67,6 +67,7 @@ class EmailTableViewCell: UITableViewCell{
     var email: Email!
     var emailState: Email.State!
     var isLoaded = false
+    var hasTurnedOnLights: Bool? = nil
```

Comment:
I think this should be handled in EmailState

Reply:
Ok, I was between those two, makes more sense to be part of the state.